paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/ErrorMessage.kt

```diff
@@ -0,0 +1,46 @@
+package com.stripe.android.paymentsheet.ui
+
+import android.graphics.Typeface
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.unit.dp
+import androidx.core.content.res.ResourcesCompat
+import com.stripe.android.uicore.StripeThemeDefaults
+import com.stripe.android.uicore.stripeTypography
+
+@Composable
+internal fun ErrorMessage(
+    error: String,
+    modifier: Modifier = Modifier,
+) {
+    val context = LocalContext.current
+    val density = LocalDensity.current
+    val typography = MaterialTheme.stripeTypography
+
+    val typeface = remember(typography) {
+        typography.fontFamily?.let {
+            ResourcesCompat.getFont(context, it)
+        } ?: Typeface.DEFAULT
+    }
+
+    val fontSize = remember(typography) {
+        with(density) {
+            val sizeInPx = StripeThemeDefaults.typography.smallFontSize.value
+            (sizeInPx.dp * typography.fontSizeMultiplier).toSp()
+        }
+    }
```

Comment:
This copies the previous implementation exactly, and is responsible for the text not actually respecting system-level font sizes. Happy to follow up with design and confirm what the behavior should be in the future.